doc/reference/reference_lua/datetime.rst

```diff
@@ -1,62 +1,964 @@
+            ---
+            - 2020-03-02T00:00:00Z
+            ...
+
+    ..  _datetime-format:
+
+    ..  method:: format( ['input_string'] )
+
+        Convert the standard ``datetime`` object presentation into a formatted string.
+        The conversion specifications are the same as in the `strftime <https://www.freebsd.org/cgi/man.cgi?query=strftime&sektion=3>`__ library.
+        Additional specification for nanoseconds is `%f` which also allows a modifier to control the output precision of fractional part: `%5f` (see the example below).
+        If no arguments are set for the method, the default conversions are used: `'%FT%T.%f%z'` (see the example below).
+
+        :param string input_string: string consisting of zero or more conversion specifications and ordinary characters
+
+        :return: string with the formatted date and time information
+        :rtype: string
+
+        **Example:**
+
+        ..  code-block:: tarantoolsession
+
+            tarantool> dt = datetime.new {
+                        nsec = 123456789,
+
+                        sec = 20,
+                        min = 25,
+                        hour = 18,
+
+                        day = 20,
+                        month = 8,
+                        year = 2021,
+
+                        tzoffset  = 180
+                        }
+            ---
+            ...
+
+            tarantool> dt:format('%d.%m.%y %H:%M:%S.%5f')
+            ---
+            - 20.08.21 18:25:20.12345
+            ...
+
+            tarantool> dt:format()
+            ---
+            - 2021-08-20T18:25:20.123456789+0300
+            ...
+
+            tarantool> dt:format('%FT%T.%f%z')
+            ---
+            - 2021-08-20T18:25:20.123456789+0300
+            ...
+
+    ..  _datetime-set:
+
+    ..  method:: set( [{ units }] )
+
+        Update the field values in the existing ``datetime`` object.
+
+        :param table units: Table of time units. The :ref:`time units <datetime-new-args>` are the same as for the ``datetime.new()`` function.
+
+        :return: updated datetime_object
+        :rtype: cdata
+
+        **Example:**
+
+        ..  code-block:: tarantoolsession
+
+            tarantool> dt = datetime.new {
+                        nsec = 123456789,
+
+                        sec = 20,
+                        min = 25,
+                        hour = 18,
+
+                        day = 20,
+                        month = 8,
+                        year = 2021,
+
+                        tzoffset  = 180
+                        }
+
+            tarantool> dt:set {msec = 567}
+            ---
+            - 2021-08-20T18:25:20.567+0300
+            ...
+
+            tarantool> dt:set {tzoffset = 60}
+            ---
+            - 2021-08-20T18:25:20.567+0100
+            ...
+
+
+    ..  _datetime-sub:
+
+    ..  method:: sub( { input[, adjust ] } )
+
+        Modify an existing datetime object by subtracting values of the input argument.
```

Comment:
Please add a link to the section "Datetime and interval arithmetic" as it contains more details about arithmetic.

Reply:
Thanks!